### PR TITLE
Require a recent profile in IUC guide lines

### PR DESCRIPTION
**Pull request by bernt-matthias**

### CODE REVIEW

File: docs/best_practices/tool_xml.rst [hunk 1]
```diff
@@ -88,6 +88,14 @@ would have a hard time finding it and gathering context about its functionality.
 With the VSearch prefix, once a user learns what one VSearch tool does, they can
 quickly apply that to the other available VSearch tools.
 
+Tool profile
+------------
+
+Tools should define a recent profile:
+
+- for new tools IUC requires the most recent Galaxy release
+- tool updates should try to use a profile not older than 2 years.
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]: What about requiring a supported Galaxy release (i.e. not older than a year) for both?

Comment (peterjc) [reply docs/best_practices/tool_xml.rst]: While well intended, a very strict policy here will hamper tool authors with an older local Galaxy server. I would therefore keep this as "should" not "must".

Comment (bernt-matthias) [reply docs/best_practices/tool_xml.rst]: > While well intended, a very strict policy here will hamper tool authors with an older local Galaxy server. I would therefore keep this as "should" not "must".

I agree. So "should" + 1 year in both cases?

File: docs/best_practices/tool_xml.rst [hunk 2]
```diff
@@ -88,6 +88,14 @@ would have a hard time finding it and gathering context about its functionality.
 With the VSearch prefix, once a user learns what one VSearch tool does, they can
 quickly apply that to the other available VSearch tools.
 
+Tool profile
+------------
+
+Tools should define a recent profile:
+
+- for new tools IUC requires the most recent Galaxy release
```
Comment (bgruening) [inline docs/best_practices/tool_xml.rst]: Is that not be too strict? I guess most Galaxy instances are lacking 1-2 version behind.

Comment (peterjc) [reply docs/best_practices/tool_xml.rst]: That seems reasonable to me, a little leeway.

Comment (bernt-matthias) [reply docs/best_practices/tool_xml.rst]: I changed it to 1 year, i.e. 3 versions and used "should".

**Review by nsoranzo — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Review by peterjc — APPROVED** [review]

**Review by wm75 — APPROVED** [review]

**Review by abretaud — APPROVED** [review]

**Pull request merged**